pages/implementation/inspector/sdk/node.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: 'guide'
+title: 'Avo Inspector Node.js SDK'
+---
+
+import Link from '../../../../components/Link';
+
+## <a name="quick-start"></a>Quick Start Guide
+
+Find the Quick Start Guide in our <a target="_blank" rel="noopener noreferrer" href="https://github.com/avohq/node-avo-inspector">Github repo.</a>
+
+## <a name="installation"></a>Installation
+
+The library is distributed with NPM, install with npm:
+
+```bash
+$ npm i node-avo-inspector
+```
+
+or yarn:
+
+```bash
+$ yarn add node-avo-inspector
+```
+
+## <a name="initialization"></a>Initialization
+
+Obtain the API key in the Inspector tab in your <a target="_blank" rel="noopener noreferrer" href="https://www.avo.app/welcome">Avo.app workspace</a>.
+
+You will need to create an instance of **AvoInspector** with the constructor:
+
+```ts
+constructor(options: {
```

Comment:
I'm feeling like there is something missing here. Do I import this constructor function from somewhere? Is options a named parameter?

Reply:
I agree, the other day was looking in web docs and thought the same about import.
The only places where we cover imports now are the quick start guide and the manage source tab.
I'll add it to all the references.

And regrading the name parameter, it's not in typescript afaik. This is how the interface generated by typescript compiler looks. Do you have any ideas how to frame it better?

What should we do regarding the

Reply:
+1 thought the same (not new in this PR so didn't think too much about it). How should we change?

Reply:
I've added the imports at least

Reply:
Hmm, I see.

In the source view in the Inspector, I feel like this is pretty clear:
![Screenshot 2020-11-30 at 12 10 19](https://user-images.githubusercontent.com/4923531/100609292-ac176700-3305-11eb-8f95-dfb43f74c71c.png)

Does it make sense to have something similar in the docs?

Reply:
The idea was to have this kinda code snippets similar to the source view in the quick start guide and here to have the interface definition 🤔
Otherwise the reference and the quick start guide would be very similar

Reply:
Ahh, I see!

Reply:
But one option might be to drop the reference and have only the quick start guide

Reply:
Yeah I think we should just go with "just show me the code" approach here. I think JS and TS developers are more used to copy-paste-able code snippets as docs rather than pure references